.github/workflows/lint.yml

```diff
@@ -74,12 +74,31 @@ jobs:
         python-version: ${{ matrix.python-version }}
     - name: Lint with pylint
       run: |
-        pip install pylint pylint-django
+        pip install pylint pylint-django textdistance
```

Comment:
I just realized this should be `pip install -r requirements/devel.txt`. I think it has been mistakenly written the first time around. In any case we don't want to be defining dependencies outside of the requirements files.